Include/opcode.h

```diff
@@ -36,6 +34,7 @@ extern "C" {
 #define NB_INPLACE_TRUE_DIVIDE                  24
 #define NB_INPLACE_XOR                          25
 
+#define NB_OPARG_LAST                           25
```

Comment:
I dislike marking the "last" of anything, you end up doing `+ 1` in various places. But I realize that it's a convention around opcode definitions and the like.